api/app/Policies/TeamPolicy.php

```diff
@@ -66,6 +77,19 @@ public function update(User $user)
      */
     public function viewAnyTeamMembers(User $user)
     {
-        return $user->isAdmin();
+        return $user->isAbleTo("view-any-teamMembers");
+    }
+
+    /**
+     * Determine whether the user can view a specific teams, team members.
+     * Likely to be updated later to allow the team admin and teammates to view their own team
+     *
+     * @param  \App\Models\User  $user
+     * @param  \App\Models\Team/null $team
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function viewTeamTeamMembers(User $user, Team $team)
```

Comment:
I would rename this to viewTeamMembers, similarly to how we name the policy function `view`, not `viewTeam`.